docs/changelog.rst

```diff
@@ -8,6 +8,17 @@ This section lists features and improvements of note in each release.
 
 The full release history can be viewed `at the GitHub perses releases page <https://github.com/choderalab/perses/releases>`_.
 
+0.9.4 - Release
+---------------
+
+Bugfixes
+^^^^^^^^
+
+- (PR `#938 <https://github.com/choderalab/perses/pull/938>`_)
+  Using new streamlined way of dealing with context caches from openmmtools.
```

Comment:
I'm presuming we'll also get the FAH PR in as well, so this should include:

```
Performance optimizations:
^^^^^^^^^^^^^^^^^^^^^^^^^
- Separate ContextCache objects are now created for propagation and energy computation in replica exchange calculations to avoid periodic cycling behavior.

Bugfixes
^^^^^^^^^
- Mixed precision and deterministic forces are used by default.
- Velocities are refreshed from the Maxwell-Boltzmann distribution each iteration to avoid sudden cooling when simulations are resumed.
- 

Enhancements
---------------
- Overhaul of Folding@home setup pipeline
- `use_given_geometry` is now `True` by default
```

Reply:
And thanks for the changelog suggestions, these look great!